trajectory:
  size: 200
  hip_knee_positions:
    '0':
      hip: 36.5584546199917
      knee: 6.14177425999860
    '1':
      hip: 36.5258869626919
      knee: 6.79724223309540
    '2':
      hip: 36.4961710866949
      knee: 7.46855086191376
    '3':
      hip: 36.4689387455844
      knee: 8.16892468182661
    '4':
      hip: 36.4407718799926
      knee: 8.90671768399806
    '5':
      hip: 36.3909442735914
      knee: 9.68422975488473
    '6':
      hip: 36.3349786277409
      knee: 10.4965571278391
    '7':
      hip: 36.2710185583195
      knee: 11.3337398311031
    '8':
      hip: 36.1942904299935
      knee: 12.1819567399985
    '9':
      hip: 36.0841569180386
      knee: 13.0188711702008
    '10':
      hip: 35.9539463082425
      knee: 13.8395527807608
    '11':
      hip: 35.8015228838333
      knee: 14.6335700263993
    '12':
      hip: 35.6228771899942
      knee: 15.3915490099996
    '13':
      hip: 35.3995725650670
      knee: 16.1046375845494
    '14':
      hip: 35.1472345364215
      knee: 16.7657028799384
    '15':
      hip: 34.8670736622759
      knee: 17.3684355919733
    '16':
      hip: 34.5587961799945
      knee: 17.9076129900006
    '17':
      hip: 34.2085079389458
      knee: 18.3745138686595
    '18':
      hip: 33.8336659951503
      knee: 18.7741154785941
    '19':
      hip: 33.4375137111026
      knee: 19.1077461061483
    '20':
      hip: 33.0210489399945
      knee: 19.3771246900006
    '21':
      hip: 32.5715951728112
      knee: 19.5805945875182
    '22':
      hip: 32.1055316424499
      knee: 19.7250198409426
    '23':
      hip: 31.6252709473153
      knee: 19.8135405149633
    '24':
      hip: 31.1309031599945
      knee: 19.8486886199999
    '25':
      hip: 30.6091618530567
      knee: 19.8279791045783
    '26':
      hip: 30.0758511197043
      knee: 19.7605538627558
    '27':
      hip: 29.5333270068100
      knee: 19.6507165538146
    '28':
      hip: 28.9819921099944
      knee: 19.5022648699989
    '29':
      hip: 28.4101029447575
      knee: 19.3140717209935
    '30':
      hip: 27.8325863221182
      knee: 19.0961700204659
    '31':
      hip: 27.2519110615462
      knee: 18.8534219382697
    '32':
      hip: 26.6686256499944
      knee: 18.5896492299979
    '33':
      hip: 26.0711938662600
      knee: 18.3029768735971
    '34':
      hip: 25.4745299248168
      knee: 18.0026470489488
    '35':
      hip: 24.8809683756228
      knee: 17.6919636962222
    '36':
      hip: 24.2906763099946
      knee: 17.3728319799972
    '37':
      hip: 23.6927554418098
      knee: 17.0405390061515
    '38':
      hip: 23.0990891638373
      knee: 16.7036517200037
    '39':
      hip: 22.5102824478370
      knee: 16.3640687980736
    '40':
      hip: 21.9250961499950
      knee: 16.0224264899968
    '41':
      hip: 21.3314412885417
      knee: 15.6728504899514
    '42':
      hip: 20.7415645286337
      knee: 15.3230860036364
    '43':
      hip: 20.1565908340237
      knee: 14.9742878927768
    '44':
      hip: 19.5760408899954
      knee: 14.6265264299967
    '45':
      hip: 18.9903581072271
      knee: 14.2732554690965
    '46':
      hip: 18.4092157130171
      knee: 13.9225186708786
    '47':
      hip: 17.8328490526804
      knee: 13.5755454080590
    '48':
      hip: 17.2599729599959
      knee: 13.2324915999968
    '49':
      hip: 16.6805630233155
      knee: 12.8871764451566
    '50':
      hip: 16.1042389483199
      knee: 12.5470544281896
    '51':
      hip: 15.5316856675629
      knee: 12.2130411123343
    '52':
      hip: 14.9624555499964
      knee: 11.8849992299969
    '53':
      hip: 14.3888398577794
      knee: 11.5564152946686
    '54':
      hip: 13.8194224197420
      knee: 11.2348462475509
    '55':
      hip: 13.2551231960129
      knee: 10.9212745545647
    '56':
      hip: 12.6957161999968
      knee: 10.6156509699970
    '57':
      hip: 12.1344143769152
      knee: 10.3120885325170
    '58':
      hip: 11.5788012383765
      knee: 10.0171145844733
    '59':
      hip: 11.0295524665810
      knee: 9.73125294440889
    '60':
      hip: 10.4862735199972
      knee: 9.45405476599713
    '61':
      hip: 9.94303487956495
      knee: 9.17934943681727
    '62':
      hip: 9.40588017125210
      knee: 8.91361120275557
    '63':
      hip: 8.87506672205492
      knee: 8.65749402511616
    '64':
      hip: 8.35001764999768
      knee: 8.41082801499722
    '65':
      hip: 7.82563566878385
      knee: 8.16806141971271
    '66':
      hip: 7.30697671962522
      knee: 7.93559153925251
    '67':
      hip: 6.79431588400027
      knee: 7.71427790111021
    '68':
      hip: 6.28729529099813
      knee: 7.50414481699721
    '69':
      hip: 5.78195788567970
      knee: 7.29993582425010
    '70':
      hip: 5.28245581558946
      knee: 7.10804358112101
    '71':
      hip: 4.78901319483776
      knee: 6.92952059134149
    '72':
      hip: 4.30127223099855
      knee: 6.76468330199706
    '73':
      hip: 3.81616057280032
      knee: 6.60835803985083
    '74':
      hip: 3.33652956766759
      knee: 6.46793931587948
    '75':
      hip: 2.86230236020604
      knee: 6.34522825221827
    '76':
      hip: 2.39305650099892
      knee: 6.24121835699675
    '77':
      hip: 1.92637769735334
      knee: 6.15131684129590
    '78':
      hip: 1.46486447898811
      knee: 6.08305963323439
    '79':
      hip: 1.00890970488672
      knee: 6.03831464843739
    '80':
      hip: 0.558757702999199
      knee: 6.01800608999632
    '81':
      hip: 0.113388653555424
      knee: 6.01661837578112
    '82':
      hip: -0.324917683235983
      knee: 6.04275150294454
    '83':
      hip: -0.755454770692153
      knee: 6.09853820040999
    '84':
      hip: -1.17769631700066
      knee: 6.18501024399579
    '85':
      hip: -1.59201942165125
      knee: 6.29617599559660
    '86':
      hip: -1.99656698932951
      knee: 6.44079294993609
    '87':
      hip: -2.39057686806332
      knee: 6.62063530080418
    '88':
      hip: -2.77334620500067
      knee: 6.83620225099518
    '89':
      hip: -3.14493779654586
      knee: 7.07981745866330
    '90':
      hip: -3.50303078213759
      knee: 7.36128171257898
    '91':
      hip: -3.84624712829857
      knee: 7.68241274498511
    '92':
      hip: -4.17320911000087
      knee: 8.04369735699445
    '93':
      hip: -4.48295254949225
      knee: 8.43634138834906
    '94':
      hip: -4.77320123439382
      knee: 8.87153242772042
    '95':
      hip: -5.04222675225082
      knee: 9.35138786271614
    '96':
      hip: -5.28827095700124
      knee: 9.87647882599359
    '97':
      hip: -5.51068961457588
      knee: 10.4366368372905
    '98':
      hip: -5.70555355054356
      knee: 11.0446400314670
    '99':
      hip: -5.87027961366940
      knee: 11.7028311002346
    '100':
      hip: -6.00254134400176
      knee: 12.4117499899926
    '101':
      hip: -6.10132904826754
      knee: 13.1596232489189
    '102':
      hip: -6.16349970118690
      knee: 13.9608410545881
    '103':
      hip: -6.18731530547213
      knee: 14.8178822368966
    '104':
      hip: -6.17147043200236
      knee: 15.7311244899916
    '105':
      hip: -6.11698949780725
      knee: 16.6869009274198
    '106':
      hip: -6.01989226155817
      knee: 17.7010015082698
    '107':
      hip: -5.87837702506547
      knee: 18.7755704491827
    '108':
      hip: -5.69098073900298
      knee: 19.9101335999908
    '109':
      hip: -5.45879422655096
      knee: 21.0884901575289
    '110':
      hip: -5.17764562464100
      knee: 22.3262729759772
    '111':
      hip: -4.84585864235300
      knee: 23.6239015477863
    '112':
      hip: -4.46236196000350
      knee: 24.9786945799903
    '113':
      hip: -4.02964557398213
      knee: 26.3710634478662
    '114':
      hip: -3.54340484267843
      knee: 27.8150619519336
    '115':
      hip: -3.00280494698663
      knee: 29.3086452521540
    '116':
      hip: -2.40799834300379
      knee: 30.8466192699904
    '117':
      hip: -1.76367524406693
      knee: 32.4058194453431
    '118':
      hip: -1.06669218199616
      knee: 33.9995956936610
    '119':
      hip: -0.318237238694559
      knee: 35.6240471163130
    '120':
      hip: 0.479186065996266
      knee: 37.2718708899912
    '121':
      hip: 1.31785503287196
      knee: 38.9172138439538
    '122':
      hip: 2.19885979987845
      knee: 40.5708965804470
    '123':
      hip: 3.11861720030574
      knee: 42.2259360535551
    '124':
      hip: 4.07239989499661
      knee: 43.8717862999925
    '125':
      hip: 5.04985530401876
      knee: 45.4792728307940
    '126':
      hip: 6.05168260368874
      knee: 47.0568470477694
    '127':
      hip: 7.07367935559648
      knee: 48.5956258940522
    '128':
      hip: 8.11088954899705
      knee: 50.0841368699940
    '129':
      hip: 9.15272065421490
      knee: 51.4927569244425
    '130':
      hip: 10.2007992256551
      knee: 52.8325434088258
    '131':
      hip: 11.2517721320347
      knee: 54.0974644964661
    '132':
      hip: 12.3017098999974
      knee: 55.2796087799951
    '133':
      hip: 13.3403112330170
      knee: 56.3538798240488
    '134':
      hip: 14.3718234039519
      knee: 57.3346020785035
    '135':
      hip: 15.3945971445947
      knee: 58.2194462395894
    '136':
      hip: 16.4061860899974
      knee: 59.0041508699956
    '137':
      hip: 17.3969946558378
      knee: 59.6669177259953
    '138':
      hip: 18.3728253617000
      knee: 60.2263714946916
    '139':
      hip: 19.3327767053871
      knee: 60.6839115591895
    '140':
      hip: 20.2748523999972
      knee: 61.0388310499954
    '141':
      hip: 21.1889708290772
      knee: 61.2725861477289
    '142':
      hip: 22.0823779848565
      knee: 61.4069573141314
    '143':
      hip: 22.9548250323991
      knee: 61.4457810951407
    '144':
      hip: 23.8049691199968
      knee: 61.3901838999950
    '145':
      hip: 24.6221764916816
      knee: 61.2232465499072
    '146':
      hip: 25.4163375812833
      knee: 60.9668874152441
    '147':
      hip: 26.1883823170488
      knee: 60.6248349759070
    '148':
      hip: 26.9377280399962
      knee: 60.1978386799945
    '149':
      hip: 27.6538440257942
      knee: 59.6684052071054
    '150':
      hip: 28.3466316076385
      knee: 59.0586252215840
    '151':
      hip: 29.0164745679838
      knee: 58.3722151443382
    '152':
      hip: 29.6621099699957
      knee: 57.6100915599942
    '153':
      hip: 30.2707373001476
      knee: 56.7554712744502
    '154':
      hip: 30.8548945711643
      knee: 55.8304435322370
    '155':
      hip: 31.4159962096031
      knee: 54.8389576796239
    '156':
      hip: 31.9537617399952
      knee: 53.7821726199940
    '157':
      hip: 32.4556892277980
      knee: 52.6440802761027
    '158':
      hip: 32.9353540299710
      knee: 51.4463632351303
    '159':
      hip: 33.3942033922436
      knee: 50.1929601042206
    '160':
      hip: 33.8314670499947
      knee: 48.8851327699940
    '161':
      hip: 34.2332489135534
      knee: 47.5074407586952
    '162':
      hip: 34.6129883155146
      knee: 46.0817143950977
    '163':
      hip: 34.9714091313702
      knee: 44.6123515476087
    '164':
      hip: 35.3072191399943
      knee: 43.1011904599940
    '165':
      hip: 35.6047444282943
      knee: 41.5342772244383
    '166':
      hip: 35.8803297446807
      knee: 39.9328554554572
    '167':
      hip: 36.1360306822593
      knee: 38.3015693342864
    '168':
      hip: 36.3716213499940
      knee: 36.6426792399938
    '169':
      hip: 36.5725351143620
      knee: 34.9430413670871
    '170':
      hip: 36.7540537352107
      knee: 33.2247018770391
    '171':
      hip: 36.9171743666872
      knee: 31.4934777124417
    '172':
      hip: 37.0603988599937
      knee: 29.7530392499933
    '173':
      hip: 37.1663435001584
      knee: 27.9924465200305
    '174':
      hip: 37.2524849654797
      knee: 26.2344874568043
    '175':
      hip: 37.3206520882035
      knee: 24.4865418449244
    '176':
      hip: 37.3704043299933
      knee: 22.7541013299923
    '177':
      hip: 37.3856366792693
      knee: 21.0280029376451
    '178':
      hip: 37.3845026620986
      knee: 19.3338773770206
    '179':
      hip: 37.3694914455047
      knee: 17.6818107234491
    '180':
      hip: 37.3406805299928
      knee: 16.0799320599908
    '181':
      hip: 37.2816823955183
      knee: 14.5222477808109
    '182':
      hip: 37.2124282205085
      knee: 13.0349018864395
    '183':
      hip: 37.1364087322952
      knee: 11.6289954911256
    '184':
      hip: 37.0546635999922
      knee: 10.3132985499892
    '185':
      hip: 36.9514272586832
      knee: 9.08187362845089
    '186':
      hip: 36.8477567569330
      knee: 7.96087981046598
    '187':
      hip: 36.7477490390010
      knee: 6.96073777900752
    '188':
      hip: 36.6527205899917
      knee: 6.08866910898828
    '189':
      hip: 36.5465334780332
      knee: 5.34150254614959
    '190':
      hip: 36.4501824937455
      knee: 4.73283268464644
    '191':
      hip: 36.3669567039978
      knee: 4.26567148054434
    '192':
      hip: 36.2969021899914
      knee: 3.94035866598931
    '193':
      hip: 36.2222024873001
      knee: 3.74308767271276
    '194':
      hip: 36.1618308602426
      knee: 3.68917497965498
    '195':
      hip: 36.1168395917721
      knee: 3.77749195286489
    '196':
      hip: 36.0849841799915
      knee: 4.00078822099296
    '197':
      hip: 36.0460286132029
      knee: 4.34239447318813
    '198':
      hip: 36.0173909187786
      knee: 4.78355758548055
    '199':
      hip: 35.9986248681916
      knee: 5.29588529026877
    '200':
      hip: 35.9863206299918
      knee: 5.84531539899867
